catalog.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header class="page-header">
+    <a class="logo" href=img alt="logo-device" src="#"></a>
```

Comment:
Ты объединил теги, а надо вложить.
```suggestion
    <a class="logo" href="#">
        <img class="logo-image" src="#">
    </a>
```